lib/msal-common/src/authority/Authority.ts

```diff
@@ -464,7 +464,11 @@ export class Authority {
      * @returns bool
      */
     static isPublicCloudAuthority(host: string): boolean {
-        return Constants.KNOWN_PUBLIC_CLOUDS.includes(host);
+        for (let index = 0; index < Constants.KNOWN_PUBLIC_CLOUDS.length; index++) {
```

Comment:
Can you do this instead?
```suggestion
        return Constants.KNOWN_PUBLIC_CLOUDS.indexOf(host) >= 0;
```